src/auto-evo/RunResults.cs

```diff
@@ -793,6 +798,11 @@ void OutputPopulationForPatch(Species species, Patch patch, long population)
                     builder.Append('\n');
             }
 
+            foreach (var entry in results.OrderByDescending(s => s.Key.PlayerSpecies).ThenBy(s => s.Key.FormattedName))
+            {
+                AddSpeciesResultString(entry.Value);
+            }
```

Comment:
Since you only use the value of the elements you iterate over in the end, I think you can directly call `dict.Values`.

This would also allow for a proper renaming of `entry` to a more explicit name.

Reply:
I agree. As far as I can tell, this entire PR could be refactored to just modify the one foreach loop that already exists with that new `OrderByDescending...` stuff and it would allow all of the other code to stay the same.

Reply:
I made the changes but I still renamed "entry" to "speciesResult". I could change it back to entry too or give it an even clearer name but I stayed at the middle ground for now.